src/library/simplified/extension/rail-cards.js

```diff
@@ -0,0 +1,11 @@
+if(utag.isRailSearchResults()) {
+ var cards = "";
+ if(b.entity.railSearch.searchParameters.cardOptions.length != undefined) 
```

Comment:
How do I get entity.railSearch.searchParameters.cardOptions to show :)? 

Here is my deeplink:
https://www.expedia.co.uk/rails/search?SearchBy=DEPARTURE&age_pax=40%2C40&dest=GBSEF&destinationName=Seaford%20(Sussex)%2C%20Seaford%2C%20UK&journeyType=oneway&orig=GB%23AS&originName=Ashford%20International%20(All%20Stations)%2C%20UK&sDate=10%2F05%2F2017&sTime=2300

And here is utag_data: 
![image](https://cloud.githubusercontent.com/assets/23199197/25836429/98f20af8-343b-11e7-849c-ef056e9c286c.png)